src/com/rcv/ResultsWriter.java

```diff
@@ -487,7 +494,7 @@ private int addHeaderRows(XSSFSheet worksheet, Map<Integer, Float> totalActiveVo
         if (rowFields[2] == OutputType.INT) {
           row.createCell(1).setCellValue((int)rowFields[1]);
         } else if (rowFields[2] == OutputType.FLOAT) {
-          row.createCell(1).setCellValue((float)rowFields[1]);
+          row.createCell(1).setCellValue((Float)rowFields[1]);
```

Comment:
For consistency!  That hobgoblin of little minds.

Reply:
Oh, interesting. I've always made a habit of using whichever one is appropriate in the immediate scope because it feels like a slightly more explicit expression of the code's intent. But I don't feel strongly about this.